content/apidocs-mxsdk/apidocs/old-version/team-server-api.md

```diff
@@ -0,0 +1,210 @@
+---
+title: "Team Server API"
+category: "API Documentation"
+menu_order: 65
+---
+
+{{% alert type="warning" %}}
+The API described on this page will be deprecated in the future. Please use the [new team server API](../team-server-api.md).
+{{% /alert %}}
+
+## 1 Introduction
+
+The team server API allows you to retrieve the information (branches, revisions) of application models stored in our team server. You always access an application model via the context of an application (see the Deploy API for more information about retrieving applications and application identifiers).
+The image below provides a domain model representation of the concepts discussed below and how these are related:
+![](attachments/131076/425989.png)
```

Comment:
The attachment file path needs to be changed, too, because the markdown file is now in a different folder. Is it possible to change the name of this old API doc to `team-server-api-version-xxx` and leave it in the same folder, instead of putting this old API doc in a new folder? In this way, we don't need to change the file path of the attachments in this old API doc.

Reply:
Is it possible to mention a version number instead of just calling it "old version"? In the future, if we have newer versions, then the current "new version" would also become an "old version", too. It would save us troubles if we now could give it a version number.

Reply:
We reverted the changes on the `Team Server API`. The new API will be called `App Repository API`. A versioning is no longer needed as now they are two different APIs.